tf-modules/archive/async_operation.tf

```
@@ -1,8 +1,9 @@
 resource "aws_cloudwatch_log_group" "async_operation" {
   name = "${var.prefix}-AsyncOperationEcsLogs"
-  retention_in_days = lookup(var.cloudwatch_log_retention_periods, "archive_async_operation_log_retention", var.default_log_retention_days)
+  retention_in_days = lookup(var.cloudwatch_log_retention_periods, "AsyncOperationTaskDefinition", var.default_log_retention_days)
```

Make sure cloudwatch log group name is used.
```suggestion
  retention_in_days = lookup(var.cloudwatch_log_retention_periods, "AsyncOperationEcsLogs", var.default_log_retention_days)
```